transactions/hybrid-custody/share_ownership.cdc

```diff
@@ -0,0 +1,22 @@
+import "HybridCustody"
+import "MetadataViews"
+
+// This transaction demonstrates how you can share ownership of one account with another, giving the receiving
+// account administrative access to the account being shared without revoking keys or removing access from the originating account.
+// This is especailly useful if you want to manage accounts from one central place, but still want to be able to do things
+// like sign in 
+transaction(shareWith: Address) {
+    prepare(acct: AuthAccount) {
+        let ownedAccount = acct.borrow<&HybridCustody.OwnedAccount>(from: HybridCustody.ChildStoragePath)
+            ?? panic("no owned account found")
+
+        let identifier = HybridCustody.getOwnerIdentifier(shareWith)
+        let path = PrivatePath(identifier: identifier)!
+
+        acct.link<&{HybridCustody.OwnedAccountPrivate, HybridCustody.OwnedAccountPublic, MetadataViews.Resolver}>(path, target: HybridCustody.ChildStoragePath)
+        let cap = acct.getCapability<&{HybridCustody.OwnedAccountPrivate, HybridCustody.OwnedAccountPublic, MetadataViews.Resolver}>(path)
+        assert(cap.check(), message: "owned account capability check failed")
+
+        acct.inbox.publish(cap, name: identifier, recipient: shareWith)
```

Comment:
Revisiting this, I'm a bit concerned about suggesting external routes of delivery for HybridCustody-related Capabilities.

Ofc, we can't prevent devs from doing this, but maybe we add a note that doing so will ambiguate resource values like `owner` in that the value cannot be taken to mean that the owner is the only one with `OwnedAccount` Capabilities.

Reply:
Definitely, I was thinking about this while writing it, as well. The main thing we're missing is that owner is a single value versus this would mean it should be an array of values. We can definitely take that on, if we want to